### DISCUSSION TITLE
tomcat -security启动之后，arthas启动失败

### DISCUSSION THREAD

**Discussion by Steven-Russell** [alibaba/arthas#2415] [Q&A]
使用tomcat的安全启动，如下所示
![image](https://user-images.githubusercontent.com/48353826/215238021-0590edcf-2f01-424b-aa97-eabba275d722.png)

然后使用arthas启动的时候，直接报错，如下所示
![arthas-failed](https://user-images.githubusercontent.com/48353826/215238207-be2fe600-9882-4bc1-8d20-3dc701dd6d7e.png)

在删除tomcat的-security参数之后，使用arthas则可以正常启动

**Comment by hengyunabc** [comment]
应该是 tomcat 启用了java的 security manager，会禁掉很多操作。arthas本身需要调不少api。

**Reply by Steven-Russell** [reply]
有没有什么在不重启tomcat或者重启java进程的方式，将这个问题规避掉？

**Reply by hengyunabc** [reply]
没有。

**Reply by Steven-Russell** [reply]
多谢
